corehq/apps/domain/templates/domain/manage_releases.html

```diff
@@ -0,0 +1,62 @@
+{% extends "hqwebapp/base_section.html" %}
+{% load hq_shared_tags %}
+{% load crispy_forms_tags %}
+{% load i18n %}
+
+{% requirejs_main 'app_manager/js/manage_releases' %}
+
+{% block page_content %}
+    {% initial_page_data 'enabled_app_releases' enabled_app_releases %}
+    {% registerurl 'location_search' domain %}
```

Comment:
is this URL used anywhere?

Reply:
was trying to add location search but could not. Would do in a follow up PR.
Removing this. thanks,